APE14.rst

```diff
@@ -0,0 +1,484 @@
+A shared Python interface for World Coordinate Systems
+------------------------------------------------------
+
+author: Thomas Robitaille, Erik Tollerud, Stuart Mumford, Adam Ginsburg
+
+date-created: 2017 November 7
+
+date-last-revised: 2017 November 7
```

Comment:
Presumably this needs updating... but I suppose it has to be on accept/reject, anyway.